pyne/utils.py

```diff
@@ -71,9 +77,12 @@ def to_sec(input_time, units):
         Time value in [sec].
 
     """
-    sec_time = input_time * time_conv_dict[units.lower()]
-    return sec_time
-
+    conv = time_conv_dict.get(units.lower(), None)
+    if conv:
+        sec_time = input_time * conv
+        return sec_time
+    else:
+        raise ValueError('Invalid units: {0}'.format(units))
```

Comment:
This is a nice addition - can you add a test for this in `test_util.py`, please?

Reply:
I expanded the test function of `utils.to_sec` for all defined units and an undefined unit (leading to `ValueError` ).